<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Const. width shapes</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>391</width>
      <height>251</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="outPath">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>C:\Users\OysteinAdm\Desktop\output.obj</string>
      </property>
     </widget>
     <widget class="QLabel" name="outPathTitle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Path:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="genShape">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>220</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Make Shape</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="xvar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="yvar">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="zvar">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_xvar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>x-var:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_yvar">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>y-var:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_zvar">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>z-var:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QProgressBar" name="progress_Task">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_taskProgress">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Task Progress:</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progress_subTask">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="labe_subTaskProgress">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sub Task Progress:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_debugText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DebugText:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="debugText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>351</width>
        <height>151</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
